setup.py

```diff
@@ -144,7 +38,6 @@ def run(self):
                 "src/kerberos_sspi.c"
             ],
         )
-    ],
-    cmdclass={"doc": doc}
```

Comment:
The README tells you to use the doc directive with setup.py to build the docs with Sphinx. Does that still work?

Reply:
No, I updated the instructions